index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dan's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section id="skeleton">
+      <header>
+        <h1 id="welcome">Welcome</h1>
+        <a href="#" id="menu">
+          <img src="images/menu.svg" alt="Drop down menu" />
```

Comment:
- [ REQUIRED]  As the last reviewer mentioned wrap your button in a nav tag because
nav is intended  for major blocks of navigation links
read this article from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)